samples/exemplar/src/main/kotlin/com/squareup/exemplar/actions/HelloWebProtoAction.kt

```diff
@@ -0,0 +1,29 @@
+package com.squareup.exemplar.actions
+
+import com.squareup.exemplar.protos.HelloWebRequest
+import com.squareup.exemplar.protos.HelloWebResponse
+import misk.security.authz.Unauthenticated
+import misk.web.Documentation
+import misk.web.Post
+import misk.web.RequestBody
+import misk.web.RequestContentType
+import misk.web.Response
+import misk.web.ResponseContentType
+import misk.web.actions.WebAction
+import misk.web.mediatype.MediaTypes
+import javax.inject.Inject
+
+class HelloWebProtoAction @Inject constructor() : WebAction {
+  @Post("/hello/proto/")
+  @Unauthenticated
+  @RequestContentType(MediaTypes.APPLICATION_PROTOBUF)
+  @ResponseContentType(MediaTypes.APPLICATION_PROTOBUF)
+  @Documentation("""
```

Comment:
`@Description` might work better.

I wonder as well if this can work for javadoc comments rather than introducing an annotation...

Reply:
I explored javadoc comments but couldn't find a way to reflect on those from within misk. We could perhaps add javadoc generation to the build process and link to the generated documentation from another admin tab?

Reply:
Unfortunately comments [don't seem to be included in the compiled class](https://stackoverflow.com/a/4135365) so annotations are the only way to go.

Reply:
i was looking at this a few weeks ago too - i don't think reflection can do it. I was looking at dokka (kotlin codedoc tool) and was imagining something like writing code doc to /resources or /build or wherever is appropriate